internal/cloudapi/v2/errors.go

```diff
@@ -44,6 +44,7 @@ const (
 	ErrorTenantNotFound               ServiceErrorCode = 28
 	ErrorNoGPGKey                     ServiceErrorCode = 29
 	ErrorValidationFailed             ServiceErrorCode = 30
+	ErrorInvalidFactsParams           ServiceErrorCode = 31
```

Comment:
The error can be dropped I think, no api changes etc...

Reply:
woops, leftover that's now removed. thank you!